### DISCUSSION TITLE
OSPO MindMap Introduction by Ana

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#35] [5️⃣ 全球化]
I think OSPO mindmap is a good project for us to join?

Anyone interested ? Need english comminication skill to colaberate with Ana from TODO group~

https://github.com/kaiyuanshe/ONES/discussions/94

Some of the task including:

- Buiding the mindmap together, enrich the content
- Chinese version translation
- Familar with the [tools ](https://markmap.js.org/)behind this map
- Build X-lab's own mindmap (such as [Opendium](https://github.com/X-lab2017/open-research/tree/main/Opendium)) using the experience of this project

![image](https://user-images.githubusercontent.com/15010826/168924374-99135ab4-caab-4d44-b75e-0206bb2d241a.png)
![image](https://user-images.githubusercontent.com/15010826/168924878-776f9427-0817-41aa-8e63-ee5e8a80c6ae.png)

**Comment by wangleo61** [comment]
申请加入贡献哈

**Comment by xgdyp** [comment]
I'm not sure what to contribute. I watched the full video and it seems not only mindmap needs help but also some CI/CD tools.

**Reply by will-ww** [reply]
Yes, many contribution points. I suggest to join the OSPO APAC meeting to discuss this opportunity.

Join the Slack to watch the meeting notification: https://thetodogroup.slack.com/archives/CATAGD6UW
